spring-cloud-azure-feature-management/README.md

```diff
@@ -151,3 +151,113 @@ Some feature filters require parameters to decide whether a feature should be tu
+
+Each of the built-in feature filters have their own parameters. Here is the list of feature filters along with examples.
+
+### PercentageFilter
+
+This filter provides the capability to enable a feature based on a set percentage.
+
+```yaml
+feature-management:
+  feature-set:
+    features:
+      feature-v:
+        enabled-for:
+          -
+            name: PercentageFilter
+            parameters:
+              percentage-filter-setting: 50
+```
+
+### TimeWindowFilter
+
+This filter provides the capability to enable a feature based on a time window. If only `time-window-filter-setting-end` is specified, the feature will be considered on until that time. If only start is specified, the feature will be considered on at all points after that time. If both are specified the feature will be considered valid between the two times.
+
+```yaml
+feature-management:
+  feature-set:
+    features:
+      feature-v:
+        enabled-for:
+          -
+            name: TimeWindowFilter
+            parameters:
+              time-window-filter-setting-start: "Wed, 01 May 2019 13:59:59 GMT",
+              time-window-filter-setting-end: "Mon, 01 July 2019 00:00:00 GMT"
+```
+
+### TargetingFilter
+
+This filter provides the capability to enable a feature for a target audience. An in-depth explanation of targeting is explained in the targeting section below. The filter parameters include an audience object which describes users, groups, and a default percentage of the user base that should have access to the feature. Each group object that is listed in the target audience must also specify what percentage of the group's members should have access. If a user is specified in the users section directly, or if the user is in the included percentage of any of the group rollouts, or if the user falls into the default rollout percentage then that user will have the feature enabled.
+
+```yml
+feature-management:
+  target:
+    enabled-for:
+      - 
+        name: targetingFilter
+        parameters:
+          users:
+            - Jeff
+            - Alicia
+          groups:
+            - 
+              name: Ring0
+              rolloutPercentage: 100
+            - 
+              name: Ring1
+              rolloutPercentage: 100
+          defaultRolloutPercentage: 50
+```
+
+## Targeting
+
+Targeting is a feature management strategy that enables developers to progressively roll out new features to their user base. The strategy is built on the concept of targeting a set of users known as the target audience. An audience is made up of specific users, groups, and a designated percentage of the entire user base. The groups that are included in the audience can be broken down further into percentages of their total members.
+
+The following steps demonstrate an example of a progressive rollout for a new 'Beta' feature:
+
+Individual users Jeff and Alicia are granted access to the Beta
```

Comment:
This should be bulleted or numbered. Right now it all runs together.

![image](https://user-images.githubusercontent.com/21000428/86965373-6dee9080-c135-11ea-9c99-24f8562bdccb.png)